tests/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ ExternalProject_Get_Property(googletest source_dir)
 set(GTEST_INCLUDE_DIRS ${source_dir}/googletest/include)
 
 ExternalProject_Get_Property(googletest binary_dir)
-set(GTEST_LIBS_DIRS ${binary_dir}/googlemock/gtest)
+set(GTEST_LIBS_DIRS ${binary_dir}/lib)
```

Comment:
Thanks a lot for that change. Since this solves a problem for now, I'm afraid it's not long-term solution. Currently we use GTest master, which means that this can change at any time. I think it'd be better to set GTest to one of the stable versions (e.g. latest version), and update it only if needed.

Reply:
I think I will actually delete this PR since it actually doesn't solve it for linux., Strangely enough in WSL it works ok.